packages/user-interface/src/components/Notification.module.scss

```diff
@@ -0,0 +1,8 @@
+.nl-portal-notification {
+  margin-block-start: var(--nlportal-space-block-2xl, 2rem);
```

Comment:
Is het handig om hier al een spacing mee te geven en niet op het moment van het plaatsen?

Reply:
In het design staan alle notificaties in de PageHeader onder de titel, dus leek mij handig om hier de correcte marge al mee te geven. Er is een optie om hier van af te wijken indien nodig door de className prop te gebruiken.

Reply:
Notificatie heb ik nu bijvoorbeeld ook in het formulier gebruikt. En het form component zelf heeft al een gap tussen de componenten. Dus dan zouden we hem hier er af moeten halen?

Reply:
Ik vraag mij af of je dit notification component binnen een form tag wilt gebruiken. In het design zie ik alle notificaties onder de titel staan. Nu worden de notifications uit de context gehaald en gerenderd in de PageHeader, zo iets generieks is lastiger als je de notificaties uit de context op verschillende plekken wilt renderen. De `scrollTo(0, 0);` kan dan in theorie ook verkeerd zijn als de notificatie niet bovenaan de pagina hoeft te staan.

Reply:
yes true, eens! normaal vind ik het wel chill om de marges niet op het component te plaatsen maar bij de implementatie maar in dit geval zie ik nu ook even geen andere situatie.